apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictions/cell-population-predictions.component.ts

```diff
@@ -98,4 +114,14 @@ export class CellPopulationPredictionsComponent {
       this.router.navigate(['/us1']);
     }
   }
+
+  /** Triggered when clicked on download CSV button  */
+  onDownloadCSVButtonClicked() {
+    const csvData = papa.unparse(this.predictions(), { header: false, columns: this.displayedColumns });
```

Comment:
Remove `header: false`. The header should be `tool,modality,...` etc. rather than the display header